pop-up.css

```diff
@@ -12,45 +12,57 @@
 }
 
 .pop-up {
-  position: fixed;
```

Comment:
- [ ] **[OPTIONAL]**  I noticed a small issue with the popup window. When I open it, I can scroll down, but the popup remains at the top. To improve the user experience, could you make it fixed so that it stays in place when scrolling?

   _Watch this video for a better understanding._
   
   [scrnli_7_29_2023_4-22-30 PM.webm](https://github.com/migcm06/miguel-portfolio/assets/107952724/2cd97bbb-649a-4dc8-bf4f-e3fca536998b)